### ISSUE TITLE
Add a link to the flutter documentation page to navigate to the related dart file

### ISSUE THREAD

**Issue by salihgueler** [flutter/flutter#24985]
**Problem**:

At the moment in the documentation page of Flutter, we do not have a button to navigate to the related dart file in Flutter project. e.g. with [FloatingActionButton](https://docs.flutter.io/flutter/material/FloatingActionButton-class.html), if we want to see the related dart file, we are not able to navigate easily. We need to go to GitHub and find the related file.

_I know we can see class related data easily but sometimes seeing the overview of the file is more helpful than usual documentation._

**Solution**:

Adding a GitHub button to navigate to related dart file would be useful.

**Comment by zoechi** [comment]
@Hixie what would be the right classification label for this issue?

**Comment by ScottS2017** [comment]
I see some of these links have been added to the https://docs.flutter.io site, and that's great!

I just happened to stumble across one that points to the wrong place. I imagine this is going to be an ongoing issue if we don't find a way to keep all of the links pointing to the correct lines in basic.dart as the file changes over time.

Currently the link button for Expanded (https://docs.flutter.io/flutter/widgets/Expanded-class.html) is pointing to line basic .dart line 3804 ( https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/basic.dart#L3804 ) when it needs to be pointing at line 4018 ( ...#L4018).

Hardcoding these links is going to incur an insane amount of technical debt, even in the short term since they're already getting outdated and 1.0 just launched a week ago. I wish I knew enough about web to have an answer, but I don't.
